config/locales/en.yml

```
@@ -53,6 +52,10 @@ en:
     new:
       header: Request an Organization Account
       description: Submit the form below if you are a pet rescue and would like your own customizable website and management platform on Homeward Tails. We'll follow up with you shortly.
+    form:
+      header: Organization Details
+      organization_name: Organization name
+      your_name: Your name
```

See other comment. This would scope under `new`
```suggestion
      form:
        header: Organization Details
        organization_name: Organization name
        your_name: Your name
```

---

ah yea you mention that here haha